cli/tests/pcluster/configure/test_pcluster_configure/test_subnet_automation_no_awsbatch_no_errors_empty_vpc/pcluster.config.ini

```diff
@@ -5,7 +5,7 @@ aws_region_name = eu-west-1
 key_name = key1
 vpc_settings = default
 # Implied value
```

Comment:
Minor: maybe the comment could be removed since schedule is no longer an implied value.

Reply:
Agreed.